test/extensions/geoip_providers/maxmind/test_data/BUILD

```diff
@@ -0,0 +1,13 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+filegroup(
+    name = "geolocation_databases",
+    srcs = glob(["*"]),
```

Comment:
Probably `glob(["*.mmdb"])` to avoid including the build file or any other irrelevant noise that might appear here.